stories/select.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import {storiesOf, addDecorator} from "@storybook/react"
+import {Select} from "@smashing/select"
+import {withA11y} from "@storybook/addon-a11y"
+import {SmashingThemeProvider} from "@smashing/theme"
+
+addDecorator(withA11y)
+
+const options = ["a", "b", "c", "d"]
+const optionsWithLabels = [
+  {value: "a", label: "A"},
+  {value: "b", label: "B"},
+  {value: "c", label: "C"},
+  {value: "d", label: "D"}
+]
+
+storiesOf("Core|Select", module)
+  .addDecorator(story => (
+    <SmashingThemeProvider theme={{}}>{story()}</SmashingThemeProvider>
+  ))
+  .add("select:default", () => (
+    <React.Fragment>
```

Comment:
Add more stories. It should work as in evergreen: https://evergreen.segment.com/components/select-menu

```jsx
    <SelectMenu
      height={300}
      options={
        ['Comedy', 'Drama', 'Fantasy', 'Family', 'Action']
          .map(label => ({ label, value: label }))
      }
      selected={state.selected}
      onSelect={item => setState({ selected: item.value })}
    >
      <Button>{state.selected || 'Select name...'}</Button>
    </SelectMenu>
```